Dockerfile

```diff
@@ -9,5 +9,6 @@ RUN mvn --batch-mode -Dmaven.test.skip=true package
 FROM eclipse-temurin:21.0.1_12-jre-alpine
 WORKDIR /app
 COPY --from=builder /workspace/target/k3a-lag-exporter-jar-with-dependencies.jar ./k3a-lag-exporter.jar
+RUN apk update && apk upgrade
```

Comment:
Should we disable caching or delete cache aftwerwards? Probably it does not matter that much. The image is gigantic because of Java anyway.